### PR TITLE
Pom changed

### PR DESCRIPTION
**Pull request by ashalya**

### CODE REVIEW

File: surveygizmo/surveygizmo-connector/surveygizmo-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 1]
```diff
@@ -196,20 +181,20 @@
...
+                                    <directory>
+                                        ${basedir}/target/tobeCopied/wso2esb-${project.version}/repository/samples/resources/security/
+                                    </directory>
+                                    <includes>
+                                        <include>**/*.jks,**/*.mar,**/synapse_sample*.xml</include>
+                                    </includes>
+                                </resource>
+                            </resources>
                         </configuration>
                     </execution>
```
Comment (kesavany) [inline surveygizmo/surveygizmo-connector/surveygizmo-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Add license header.Format the code.

Comment (ashalya) [inline surveygizmo/surveygizmo-connector/surveygizmo-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: fixed in https://github.com/wso2/esb-connectors/pull/432

File: surveygizmo/pom.xml [hunk 2]
```diff
@@ -1,190 +0,0 @@
...
-					<groupId>javax.servlet</groupId>
-					<artifactId>servlet-api</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
-
-		<dependency>
-			<groupId>org.wso2.carbon</groupId>
-			<artifactId>org.wso2.carbon.integration.framework</artifactId>
-			<version>4.0.0</version>
```
Comment (msmshariq) [inline surveygizmo/pom.xml]: It's better to the version as a property and use it here, that makes it easier to manage dependencies.

Comment (ashalya) [inline surveygizmo/pom.xml]: Fixed in latest code. Moved all the dependency to parent pom

**Pull request merged**